content/articles/cancer-cell-classification-using-scikit-learn/index.md

```diff
@@ -0,0 +1,155 @@
+Before we can learn how to train our model, we need to organize the data and and if you wish to what it contains then you can use the **print()** function in order to gain a better sense of what we have.
+
+```python
+namesForlabels = data['target_names']
+ourlabels = data['target']
+namesForfeature = data['feature_names']
+ourfeatures = data['data']
+```
+You can then use the **print()** method to examine the data.
+
+```python
+print(namesForlabels)
+```
+Output:
+
+```bash
+['malignant' 'benign']
+```
+Tumor data is classified as `malignant` or `non-malignant` as you can see.
+
+Labels 0 and 1 are binary values representing labels where 0 indicates malignant tumors and 1 indicates benign tumors when they are written.
+
+```python
+print(ourlabels)
+```
+The tumor's 30 distinct characteristics and properties may be seen in full detail when the features are printed. Training our model on factors such as a tumor's malignant or benign status will help us make an accurate prediction.
+
+```python
+print(ourfeatures)
+```
+There are 569 instances of tumor data in the dataset in question, each with a numerical value for one of the 30 variables in question. Data you have acquired indicates that the first tumor is malignant and has a mean radius of 1.79900000e+01 based on the information you know.
+
+#### 4. Creating sets of data to organize the information
+We need to evaluate our classifier's accuracy by running it on data that has never been seen before.This means we'll divide our data into two sets, the training and test sets, before we begin creating our model. Training and evaluation of the model will be done on the training set, and then it will be used to predict the unknown test set.
+
+Sklearn comes with a function called **train test split()**, which automatically divides the data into these two groups for us.
+
+```python 
+from sklearn.model_selection import train_test_split
+
+train, test, trainingLabels, testingLabels = train_test_split(ourfeatures, ourlabels, test_size = 0.33, random_state = 42)
+```
+It is possible to utilize the test size as a parameter to randomly divide the data. We've divided the original data into test data by a third of its original size. Other than that, the rest of the data is used for training. A different label for the train and test variables is used for each.
+
+Refer to the [documentation](https://scikit-learn.org/stable/modules/generated/sklearn.model_selection.train_test_split.html) for further information on how to use **train_test_split**() function.
+
+#### 5. Building the Model
+You can select from a wide variety of machine learning models to meet your needs. Each one has advantages and cons of its own. A popular method for binary classification, dubbed `Naive Bayes,` has been selected for use in this model. GaussianNB is the first module that needs to be imported and initialized using **GaussianNB()** function. then use the **fit()** method to train your model on the dataset's data.

```

Comment:
We cannot start a sentence with, "then use the fit()".  Please make that correction